src/constants/enums/processStages.ts

```diff
@@ -8,12 +8,21 @@ const processStages = [
     name: 'I have done some initial research'
   },
   {
-    value: 'funded',
-    name: 'The project is already funded'
+    value: 'inDevelopment',
+    name: 'Initial development underway'
   },
   {
-    value: 'inDevelopment',
-    name: 'The project is already in development'
+    value: 'inOM',
+    name: 'I am in Operations & Maintenance'
+  },
+  {
+    value: 'inOMProposal',
+    name:
+      'I am in Operations & Maintenance but will be soliciting proposals for a different direction'
+  },
+  {
+    value: 'servicesContract',
+    name: 'This project is a services contract'
```

Comment:
My understanding is that we're not doing any mapping of old fields to these new fields.

However, should we be doing mapping of a removed field `funded` to a new field? What happens when an Intake is in the `funded` status, and we try and render it?

Reply:
Ah, that's true.  The mapping, if it's going to happen, would fall outside the scope of this ticket, so best to not remove that field.